### PR TITLE
Hotel theming and re-run update script

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR add's the bulk of the theming for the hotel content type. Some fields from the existing ecms_hotels feature package had to be updated to support the design.

Also included is an update to to update hook from the last deployment since it didn't run.

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | yno
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: ecms_base/modules/custom/ecms_blocks/src/Plugin/Block/HotelListingBlock.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    // Return a list of rendered teaser nodes.
+    $builder = $this->entityTypeManager->getViewBuilder('node');
+    return $builder->viewMultiple($nodes, 'teaser', $language);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function blockAccess(AccountInterface $account): object {
```
Comment (ben-hamelin) [inline ecms_base/modules/custom/ecms_blocks/src/Plugin/Block/HotelListingBlock.php:125]: return type should be `AccessResult`

**Review by ben-hamelin — CHANGES_REQUESTED** [review]
Overall this looks good. One return type update reqeusted.
Only other thought here would be if it is more appropriate to have this custom block within the "hotel" feature? That would make it directly dependent on that module being installed, keeping it out of the available block list if it was not.

**Comment by jordan-caldwell** [comment]
> Overall this looks good. One return type update reqeusted.
> Only other thought here would be if it is more appropriate to have this custom block within the "hotel" feature? That would make it directly dependent on that module being installed, keeping it out of the available block list if it was not.

I asked @pfrilling when I was creating this block and he suggested keeping it with the other ecms_blocks. I don't have a strong preference either way but I was concerned that the feature might overwrite some of the custom code. I did not test that out though.

**Comment by ben-hamelin** [comment]
@jordan-caldwell Ok that's fine. The guard clause is in place so I think that covers us.
/approved

**Pull request merged**